SiteGenerator.ConsoleApp/HandlebarsConverter.cs

```diff
@@ -70,13 +69,13 @@ private void IncludeHelper(TextWriter writer, dynamic context, object[] paramete
             writer.WriteSafeString(result);
         }
 
-        private static void MarkdownHelper(TextWriter writer, HelperOptions options, dynamic context,
+        private void MarkdownHelper(TextWriter writer, HelperOptions options, dynamic context,
             object[] arguments)
         {
             var stringWriter = new StringWriter();
             options.Template(stringWriter, context);
 
-            string html = MarkdownConverter.ToHtml(stringWriter.ToString());
+            string html = MarkdownConverter.ToHtml(stringWriter.ToString(), topLevelConfig.Config.LineBreaks!.Value);
```

Comment:
The `LineBreaks!.Value` pattern is clearly an antipattern. We could make the `LineBreaks` setting in `topLevelConfig.Config` be non-nullable (which is the default for value types like enums), but the problem with doing so is that YamlDotNet will then use the first entry in the enum as the value when deserializing. But, I guess there would be ways around this. Living with this for now.